tensorflow/lite/micro/examples/mnist_lstm/train.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+# pylint: disable=line-too-long
```

Comment:
Do we need this?

Reply:
I saw every file has this so I included it as well. Should I remove it

Reply:
Yes it just disables the linter message, which isn't needed here.

Reply:
Oh you mean the pylint command. I thought you want me to remove the copyright banner. Yeah I added this so my IDE wont complain anymore. I will remove it. This is my first PR. Thank you so much for pointing out this!